src/condor_ce_trace

```diff
@@ -316,7 +329,7 @@ def main():
         raise ce.CondorRunException('ERROR: Could not find CE schedd at %s.\n' % job_info['schedd_name'] + \
                                     'Verify that the Scheduler daemon is up with `condor_ce_status -any`.')
 
-    configure_auth()
+    os.environ.setdefault('_condor_SEC_CLIENT_AUTHENTICATION_METHODS', 'SCITOKENS,GSI,FS')
```

Comment:
If `_condor_SEC_CLIENT_AUTHENTICATION_METHODS` is already set then `os.environ.setdefault()` will not update it with the new value. Do you care?

Reply:
Yeah, that's what we want, so users can override the method list if they'd like